Backpack/Icon/Classes/BPKIconView.h

```diff
@@ -45,6 +45,7 @@ NS_ASSUME_NONNULL_BEGIN
 @property(nullable, nonatomic, copy) BPKIconName iconName;
 
 /**
+ * DEPRECATED. THIS WILL BE REMOVED SOON AS SETTING SIZE DURING LIFECYCLE WILL BE UNSUPORTED.
```

Comment:
There's a blessed way to mark things deperacted, we should use that?

Reply:
Gonna see if that causes issues in the main app.